docs/api/python/data_ingestion.md

```diff
@@ -112,3 +112,87 @@ The sample size can be changed by setting the `pandas_analyze_sample` config opt
 # example setting the sample size to 100000
 duckdb.default_connection.execute("SET GLOBAL pandas_analyze_sample=100000")
 ```
+
+### Object Conversion
+
+This is a mapping of python object type to DuckDB logical type:
+
+`None` -> `NULL`  
+`bool` -> `BOOLEAN`  
+
+#### `int`
+Since integers can be of arbitrary size in Python, there is not a one-to-one conversion possible for ints.  
+Intead we perform these casts in order until one succeeds:  
+- `BIGINT`
```

Comment:
is hugeint going to be tried once its in?

Reply:
Hugeint has been in for a while, it's not tried in this cast yet though.
When this was implemented it seemed too complex

No one has complained about it yet, when they do I'll have another look